### PR TITLE
Add arewewebyet team

### PR DESCRIPTION
**Pull request by giraffate**
https://github.com/rust-lang/arewewebyet/issues/397#issuecomment-1484086231

Add a team responsible for https://www.arewewebyet.org/ and https://github.com/rust-lang/arewewebyet

### CODE REVIEW

File: teams/arewewebyet.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name = "arewewebyet"
```
Comment (rylev) [inline teams/arewewebyet.toml:1]: This should be a subteam of dev-tools like [this](https://github.com/rust-lang/team/blob/2d25539ba2b08252d629373cb40851d676939f7a/teams/cargo.toml#L2). cc @Manishearth

**Review by rylev — COMMENTED** [review]
Waiting for approval from @Manishearth as to whether we really want to add this as a new team or not.

**Comment by Manishearth** [comment]
This is definitely not a subteam of devtools.

**Comment by Manishearth** [comment]
You'll need to figure out where it goes, but it's not devtools.

**Comment by rylev** [comment]
I think then we'll want this to be a marker team in much the same way [the emacs team](https://github.com/rust-lang/team/blob/03de88a2b11edee98496181686b58a7355d360b1/teams/emacs.toml#L2) works.

**Comment by giraffate** [comment]
Thanks! I addressed it.

**Comment by giraffate** [comment]
friendly ping @Manishearth @rylev

**Comment by Manishearth** [comment]
(I am not able to merge this)

**Pull request merged**